fingerprint_verification.py

```diff
@@ -1,44 +1,148 @@
 ######################################################################
 # Names of any others you worked with:
+# Received some help/advice from Hailee & Bryant (Namely my trouble with calling the correct file, then with some of the math for shifted_check)
+# Additionally, some use of reddit/google & my notes for certain built-ins (i.e. splitlines(), len, etc.)
 # AI transcript if used:
 # Any extensions done:
 ######################################################################
 
+# open and read the file then return the content as a string
 def read_print(filename):
     """Reads the given print and returns a dictionary with all info."""
-    # Your turn!
-
-
-
-
-
-
-# You can add your various check functions below
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+    with open(filename, 'r') as fh:
+        # splitlines after read to remove \n
+        lines = fh.read().splitlines()
+    # the string of characters being the fingerprint
+    # currently the "name, width, height" comes according to file
+        name = lines[0]
+        width = lines[1]
+        height = lines[2]
+        fingerprint = lines[3:]
+    # add to a dictionary
+        data = {'name': name, 'width': width, 'height': height,
+                'fingerprint': fingerprint}
+        return data
+
+# we just want to show a dictionary at the end (like a parsed version of the file)
+# showing the comparisons
+
+# simple check if match True, if not False
+def simple_check(fingerprint1, fingerprint2):
+    if fingerprint1 != fingerprint2:
+        return False
+    return True
+
+# variant check, if matches/total greater/equal to threshold, True, if not False
+def variant_check(fingerprint1, fingerprint2, threshold = .95):
+    # matches count number of matching characters with our fingerprints
```

Comment:
Your documentation here is generally great. I'd just recommend trying to use Python's docstring syntax to put some of this immediately after the `def` header line and in the triple quote syntax. And generally address what types of things can be arguments to your function, and what types of things it returns (if any)